ui/packages/audio/src/Audio.svelte

```diff
@@ -157,7 +157,7 @@
 	{#if source === "microphone"}
 		<div class="mt-6 p-2">
 			{#if recording}
-				<button class="gr-button !bg-red-500/10" on:click={stop}>
+				<button class="gr-button !bg-red-500/10" style={style["record_btn"]} on:click={stop}>
```

Comment:
Similar to the comments above, we don't want to just pass _everything_ down to the element. We want an explicit interface like `label_bg` etc. It would be really easy to break layouts with arbitrary CSS. I would just make the record available to each component and let it decide how the CSS is handled. 

Keeping this interface constrained and gradually opening it up makes more sense to me in terms of ensuring gradio Apps always look good. Theming should be where we really let people customise the general look and feel (when we figure out how we'll be doing that).